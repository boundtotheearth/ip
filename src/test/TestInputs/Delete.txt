todo read book
todo read novel
delete 1
list
