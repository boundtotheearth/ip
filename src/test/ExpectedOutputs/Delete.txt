Added: [T][✘] read book
Added: [T][✘] read novel
I've removed this task:
[T][✘] read book
1. [T][✘] read novel
